conduit/plugins/exporters/postgresql/README.md

```diff
@@ -19,26 +19,34 @@ For additional details, refer to the [parsing documentation here](https://pkg.go
 
 The delete-task prunes old transactions according to its configuration. This can be used to limit the size of the database.
 
+### Pruning an Existing Database
+
+The first run after enabling pruning for an existing database must delete all of the older transactions. For many configurations, this process may take many hours or days. While the initial delete is in progress, the transaction table will be locked, and you'll see database errors in the log.
```

Comment:
```suggestion
The first run after enabling pruning for an existing (already populated) database must delete all of the older transactions. For some configurations, this process may take many hours or days. While the initial delete is in progress, the transaction table will be locked, and you'll see database errors in the log.
```

Reply:
`existing` and `already populated` are trying to say the same thing. How about this:
```suggestion
The first run after enabling pruning for a pre-existing database must delete all of the older transactions. This process may take many hours. While the initial delete is in progress, the transaction table will be locked, and you'll see database errors in the log.
```

Reply:
ok I'm fine with that, just a nit anyway. I think "existing" to me doesn't imply that it necessarily already has data in it.

Reply:
Great, thanks Anne. It also refers to "all of the older transactions", so hopefully people recognize it as a complete idea.